src/pages/workspace/rules/ExpenseReportRulesSection.tsx

```diff
@@ -0,0 +1,193 @@
+            onToggle: (isEnabled: boolean) => {
+                PolicyActions.enableAutoApprovalOptions(policyID, isEnabled);
+            },
+            subMenuItems: [
+                <OfflineWithFeedback pendingAction={policy?.autoApproval?.pendingFields?.limit}>
+                    <MenuItemWithTopDescription
+                        key="autoApproveReportsUnder"
+                        description={translate('workspace.rules.expenseReportRules.autoApproveReportsUnderTitle')}
+                        title={CurrencyUtils.convertToDisplayString(
+                            policy?.autoApproval?.limit ?? CONST.POLICY.AUTO_APPROVE_REPORTS_UNDER_DEFAULT_CENTS,
+                            policy?.outputCurrency ?? CONST.CURRENCY.USD,
+                        )}
+                        shouldShowRightIcon
+                        style={[styles.sectionMenuItemTopDescription, styles.mt6, styles.mbn3]}
+                        onPress={() => Navigation.navigate(ROUTES.RULES_AUTO_APPROVE_REPORTS_UNDER.getRoute(policyID))}
+                    />
+                </OfflineWithFeedback>,
+                <OfflineWithFeedback pendingAction={policy?.autoApproval?.pendingFields?.auditRate}>
+                    <MenuItemWithTopDescription
+                        key="randomReportAuditTitle"
+                        description={translate('workspace.rules.expenseReportRules.randomReportAuditTitle')}
+                        title={`${policy?.autoApproval?.auditRate ?? CONST.POLICY.RANDOM_AUDIT_DEFAULT_PERCENTAGE}%`}
+                        shouldShowRightIcon
+                        style={[styles.sectionMenuItemTopDescription, styles.mt6, styles.mbn3]}
+                        onPress={() => Navigation.navigate(ROUTES.RULES_RANDOM_REPORT_AUDIT.getRoute(policyID))}
+                    />
+                </OfflineWithFeedback>,
+            ],
+        },
+        {
+            title: translate('workspace.rules.expenseReportRules.autoPayApprovedReportsTitle'),
+            subtitle: autoPayApprovedReportsUnavailable
+                ? renderFallbackSubtitle(translate('common.payments').toLowerCase())
+                : translate('workspace.rules.expenseReportRules.autoPayApprovedReportsSubtitle'),
+            switchAccessibilityLabel: translate('workspace.rules.expenseReportRules.autoPayApprovedReportsTitle'),
+            onToggle: (isEnabled: boolean) => {
+                PolicyActions.enablePolicyAutoReimbursementLimit(policyID, isEnabled);
+            },
+            disabled: autoPayApprovedReportsUnavailable,
+            showLockIcon: autoPayApprovedReportsUnavailable,
+            isActive: policy?.shouldShowAutoReimbursementLimitOption && !autoPayApprovedReportsUnavailable,
+            pendingAction: policy?.pendingFields?.shouldShowAutoReimbursementLimitOption,
+            subMenuItems: [
+                <OfflineWithFeedback pendingAction={policy?.autoReimbursement?.pendingFields?.limit}>
```

Comment:
We only have one sub-option, so let's use the pendingFields?.autoReimbursement to match with optimistic data 

```suggestion
                <OfflineWithFeedback pendingAction={policy?.pendingFields?.autoReimbursement}>
```

Reply:
this is the same thing as above - if you approach it this way, you're treating them as a group and this might result in graying out the fields that aren't being processed offline